packages/integrations/mdx/README.md

```diff
@@ -518,6 +518,12 @@ These are plugins that modify the output [estree](https://github.com/estree/estr
 
 We suggest [using AST Explorer](https://astexplorer.net/) to play with estree outputs, and trying [`estree-util-visit`](https://unifiedjs.com/explore/package/estree-util-visit/) for searching across JavaScript nodes.
 
+### remarkRehype
+
+Options for `remarkRehype`. See [markdown.remarkRehype](https://docs.astro.build/en/reference/configuration-reference/#markdownremarkrehype).
+
+When `extendPlugins` is set to `'markdown'` (default), this config is extended with `remarkRehype` from Astro's default markdown config.
```

Comment:
```suggestion
### remarkRehype

Markdown content is transformed into HTML through remark-rehype which has [a number of options](https://github.com/remarkjs/remark-rehype#options).

You can use remark-rehype options in your MDX integration config file like so:

```js
// astro.config.mjs
export default {
  integrations: [mdx({
    remarkRehype: {
      footnoteLabel: 'Catatan kaki',
      footnoteBackLabel: 'Kembali ke konten',
    },
  })],
};
```
This inherits the configuration of [`markdown.remarkRehype`](https://docs.astro.build/en/reference/configuration-reference/#markdownremarkrehype). This behavior can be changed by configuring `extendPlugins`.
```
Going by what the Markdown config example looks like, what about sticking with a similar format, which is also in keeping with the remark and rehype plugin options? How does something along these lines look?  (Edit to taste.)

Reply:
Thanks @sarah11918! Change looks good to me 👍

Reply:
I have re-added the last line as GitHub dropped it together with the closing ``` for some reason.